### DISCUSSION TITLE
Sound stream generation

### DISCUSSION THREAD

**Discussion by ghost** [Hanziness/AnotherPomodoro#185] [Ideas & feature requests]
Generate sounds like [Noisli](https://www.noisli.com) and similar programs and services. Quite appropriate, at least in the long run.

**Comment by Hanziness** [comment]
I'm afraid even the integration of Noisli or MyNoise is beyond the scope of this application (and I wouldn't want to bloat it with too many features, anyway) :/

Maybe some tips & tricks section could include links to these services to help users improve their productivity.

**Reply by ghost** [reply]
I also thought that it was more suitable for the fork.
